packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -194,6 +221,7 @@ Each LogEvent object describes a single logged action or "event" that is perform
               "source": String, Optional
           ], Optional
 }
```

Comment:
This closing bracket should be indented to be on the same level with `request` field.

![image](https://github.com/okta/okta-developer-docs/assets/32918538/671fd548-7055-4298-84a6-4ddb436d8653)